frontend/src/home/Home.tsx

```diff
@@ -119,7 +119,6 @@ export const Inner: FC<{ categories: Record<string, Tag> }> = ({
 export const Home: FC = () => {
     const { data } = useSWR('/api/tags', async () => {
         const request = await fetch(GLOBALS.API_URL + '/tags/list');
-
```

Comment:
Doesn't eslint force a enter before the return statement?

Reply:
`pnpm lint` didn't, VSCode ext did.